group_vars/all/mysql.yml

```diff
@@ -0,0 +1,11 @@
+---
+mysql_root_password:
```

Comment:
What happens when you set this to null?

Reply:
Ooops that's not intendet. See 39ea38be8067467d510675f6a33ed9d76d4f13ed